basicOptions.lua

```diff
@@ -318,6 +318,10 @@ fctfloatmodeDropdown.initialize = function(dropdown)
 		info.func = function(self)
 			addon:SetCVar("floatingCombatTextFloatMode", self.value)
 			UIDropDownMenu_SetSelectedValue(dropdown, self.value)
+			if not InCombatLockdown() then
```

Comment:
Ow, I still managed to open the options window somehow, but removed that unneeded conditional now